### PR TITLE
James 1844 2

### PR DESCRIPTION
**Pull request by Laura-Royet**

### CODE REVIEW

**Comment by Laura-Royet** [comment]
capture d ecran de 2016-10-25 15 00 45
capture d ecran de 2016-10-25 15 01 15
capture d ecran de 2016-10-25 15 01 33

File: src/site/xdoc/contribute.xml [hunk 1]
```diff
@@ -39,63 +39,53 @@
...
-        <br />
-        Subscribe to the relevant mailing lists (link on the left).
+      </p>
+    </section>
+
+    <section name="How to contribute to the code?">
+      <p>
+        Clone the source code of the project from its 
+        <a href="https://github.com/apache/james-project">Github</a>
         <br />
```
Comment (chibenwa) [inline src/site/xdoc/contribute.xml]:
```
or its <a href="git://git.apache.org/james-project.git">apache git repository</a>
```

File: src/site/xdoc/contribute.xml [hunk 2]
```diff
@@ -39,63 +39,53 @@
...
+        <a href="https://james.apache.org/mail.html">mailing lists</a>
         <br />
-        <a href="http://jakarta.apache.org/site/guidelines.html">A good, full, summary of links to guidelines</a>
-        <br />
-        Subscribe to the relevant mailing lists (link on the left).
+      </p>
+    </section>
+
+    <section name="How to contribute to the code?">
+      <p>
```
Comment (chibenwa) [inline src/site/xdoc/contribute.xml]: We should explain first how to create a ticket...

File: src/site/xdoc/contribute.xml [hunk 3]
```diff
@@ -39,63 +39,53 @@
...
+        Clone the source code of the project from its 
+        <a href="https://github.com/apache/james-project">Github</a>
         <br />
-        <a href="http://jakarta.apache.org/site/contributing.html">Craig R. McClanahan's advice how to get involved</a>
+        Create a Pull Request with your branch name
+        and prefix its different commits with this name.
```
Comment (chibenwa) [inline src/site/xdoc/contribute.xml]: with ticket number.

File: src/site/xdoc/contribute.xml [hunk 4]
```diff
@@ -39,63 +39,53 @@
...
         <br />
-        Make your patch from the jakarta-james directory and make sure
-        it conforms
-        to the code standards, otherwise it may be ignored. It is OK to make a
+        Make sure it conforms to the code standards, otherwise it may be ignored. It is OK to make a
         single patch covering several
         files, but please only one issue at a time.
         <br />
-        Prefix the mail subject with [PATCH]
+        Prefix the JIRA issue subject with [PATCH]
```
Comment (chibenwa) [inline src/site/xdoc/contribute.xml]: We can remove this line about sending patch by mail ?

File: src/site/xdoc/contribute.xml [hunk 5]
```diff
@@ -39,63 +39,53 @@
...
       </p>
 
       <p>
         The reason for these rules is so that commiters can easily see
         what you are trying to achieve,
-        it is their resonsibility to manage the code and review submissions,
+        it is their responsibility to manage the code and review submissions,
         if you make it easy for them to see what you are doing your
         patch is more likely to
         be commited quickly (or at all).
```
Comment (chibenwa) [inline src/site/xdoc/contribute.xml]: Can we remove "at all" ?

File: src/site/xdoc/contribute.xml [hunk 6]
```diff
@@ -105,7 +95,7 @@
 
     <section name="Adding New Code">
       <p>
-        Like the principles for patch submission, mark your mail [PATCH]
+        Like the principles for patch submission, mark your JIRA issue subject [PATCH]
```
Comment (chibenwa) [inline src/site/xdoc/contribute.xml]: Is this section adding new code needed at all ?

File: src/site/xdoc/contribute.xml [hunk 7]
```diff
@@ -117,35 +107,55 @@
...
         <br />
         It also helps if you send your files in an archive (tar, zip)
-        which preserves directories, make it from the jakarta-james
+        which preserves directories make it from james-project
         directory so we can un-tar your files into the right place.
       </p>
     </section>
 
-    <section name="Reporting and Fixing Bugs">
+    <section name="Reporting Bugs">
```
Comment (chibenwa) [inline src/site/xdoc/contribute.xml]: This should be explained before contributing to code :-)

File: src/site/xdoc/contribute.xml [hunk 8]
```diff
@@ -117,35 +107,55 @@
...
+        different fields as accurately as possible, so that 
+        any user can reproduce the reported bug. 
+        Also note that all your information must be readable 
+        (use markedown).
+        <br />
+        Then, you have to click on Create to submit your bug.
+      </p>
+    </section>
+
+    <section name="Fixing Bugs">
```
Comment (chibenwa) [inline src/site/xdoc/contribute.xml]: Is this section needed ?

File: src/site/xdoc/contribute.xml (hunk 3 above)
Comment (Laura-Royet) [inline src/site/xdoc/contribute.xml]: done

File: src/site/xdoc/contribute.xml (hunk 5 above)
Comment (Laura-Royet) [inline src/site/xdoc/contribute.xml]: ok, removed.

File: src/site/xdoc/contribute.xml (hunk 6 above)
Comment (Laura-Royet) [inline src/site/xdoc/contribute.xml]: ok, removed

File: src/site/xdoc/contribute.xml (hunk 7 above)
Comment (Laura-Royet) [inline src/site/xdoc/contribute.xml]: ok, moved at the appropriate place.

File: src/site/xdoc/contribute.xml (hunk 8 above)
Comment (Laura-Royet) [inline src/site/xdoc/contribute.xml]: No, you are right.
I remove it.

**Comment by Laura-Royet** [comment]
capture d ecran de 2016-10-26 10 53 57
capture d ecran de 2016-10-26 10 54 20

File: src/site/xdoc/contribute.xml [hunk 9]
```diff
@@ -45,13 +45,53 @@
...
+
     <section name="How to contribute to the code?">
       <p>
         Clone the source code of the project from its 
         <a href="https://github.com/apache/james-project">Github</a>
+        or its <a href="git://git.apache.org/james-project.git">apache git repository</a>
         <br />
-        Create a Pull Request with your branch name
-        and prefix its different commits with this name.
+        Create your branch and name it with the JIRA ticket number.
```
Comment (chibenwa) [inline src/site/xdoc/contribute.xml]: Create your branch and prefix it with the JIRA ticket ?

**Pull request merged**